nd4j/nd4j-serde/nd4j-arrow/src/main/java/org/nd4j/arrow/ArrowSerde.java

```diff
@@ -122,11 +126,21 @@ public static void addTypeTypeRelativeToNDArray(FlatBufferBuilder bufferBuilder,
                 Tensor.addTypeType(bufferBuilder,Type.Int);
                 break;
             case FLOAT:
-                Tensor.addTypeType(bufferBuilder,Type.FloatingPoint);
+                Tensor.addTypeType(bufferBuilder, Type.FloatingPoint);
                 break;
             case DOUBLE:
                 Tensor.addTypeType(bufferBuilder,Type.Decimal);
                 break;
+            case HALF:
```

Comment:
We probably want to add other types here as well: bfloat16, LONG and unsigned types?

Reply:
I don't think Arrow supports bfloat16. What do we do in those cases @AlexDBlack?

Reply:
Yeah, it only has float16/32/64, no bfloat16
https://arrow.apache.org/docs/metadata.html
In those cases we'll need to convert, probably to yloat32 in the case of BFloat16

Reply:
Ye, bfloat16 range is the same as float32. float16 range is way smaller.

Reply:
@raver119 what do you think here? This was the official arrow bindings. I can migrate this over to the new one. See ByteDecoArrowSerde now.